### PR TITLE
Add stop-words.json

### PR DESCRIPTION
**Pull request by rmzelle**
Updates the stop words that should be recognized by CSL processors, and places them in a separate JSON file.

See http://xbiblio-devel.2463403.n2.nabble.com/title-casing-skip-words-td7578629.html and https://gist.github.com/adam3smith/6326169 .

I'll accept this pull request once people have had a chance to review.

### CODE REVIEW

**Comment by bdarcus** [comment]
Bumping.

Obviously if we merge, the merge conflict would need to be resolved. I think a `get merge master` will fix it, as it's probably just because I changed the file extension.

File: specification.txt [hunk 1]
```diff
@@ -2373,9 +2373,11 @@ items is performed by:
    is capitalized. The case of words in mixed or uppercase stays the same.
    
 In both cases, stop words are lowercased, unless they are the first or last word
-in the string, or follow a colon. The stop words are "a", "an", "and", "as",
-"at", "but", "by", "down", "for", "from", "in", "into", "nor", "of", "on",
-"onto", "or", "over", "so", "the", "till", "to", "up", "via", "with", and "yet".
+in the string, or follow a colon. The stop words are listed in the file
+`stop-words.json
+<https://raw.github.com/citation-style-language/documentation/master/stop-words.json>`_.
```
Comment (rmzelle) [inline specification.txt:2378]: Per https://github.com/citation-style-language/schema/issues/216#issuecomment-638415275, this should probably become something like https://resource.citationstyles.org/.../stop-words.json.

Comment (bdarcus) [inline specification.txt:2378]:
```suggestion
<http://resource.citationstyles.org/schema/latest/input/json/stop-words.json>`_.
```

Comment (bdarcus) [reply specification.txt:2378]: We want the json file in the schema repo; right?

I put it under `/schemas/styles/stop-words.json`.

Is that what you were thinking?

If yes, need to get the redirect to work.

Comment (bdarcus) [reply specification.txt:2378]: Or did you want to keep it here?

Comment (bwiernik) [reply specification.txt:2378]: We should serve the file stored at `/schemas/styles/stop-words.json` out of a link from `https://resource.citationstyles.org/`

Comment (bdarcus) [reply specification.txt:2378]: > We should serve the file stored at `/schemas/styles/stop-words.json` out of a link from `https://resource.citationstyles.org/`

@dstillman - could you please set up the redirect so we can point to the styles schema files; for example, this one?

https://github.com/citation-style-language/schema/blob/master/schemas/styles/stop-words.json

Comment (bwiernik) [reply specification.txt:2378]: So, that would specifically be any URL with the pattern `https://resource.citationstyles.org/schemas/(.*)` redirects to `https://github.com/citation-style-language/schema/blob/master/schemas/(.*)` ?

Comment (dstillman) [reply specification.txt:2378]: We're currently redirecting `schema/latest/input/json/` to `citation-style-language/schema/master/schemas/input/`.

Should that remain, or should it switch solely to `schemas` (plural) URLs?

Comment (bdarcus) [reply specification.txt:2378]: > We're currently redirecting `schema/latest/input/json/` to `citation-style-language/schema/master/schemas/input/`.
> 
> Should that remain, or should it switch solely to `schemas` (plural) URLs?

The current should remain; we just need to also be able to access the `styles` sibling directory.

Comment (dstillman) [reply specification.txt:2378]: I just need the exact path to redirect.

Comment (bdarcus) [reply specification.txt:2378]: You can't use @bwiernik's suggestion to redirect the common parent directory; 

`https://resource.citationstyles.org/schemas/(.*)` to 
`https:///raw.githubusercontent.com/citation-style-language/schema//master/schemas/(.*)`?

If not, would be this specific file redirect:

`https://resource.citationstyles.org/schemas/styles/stop-words.json` to
`https://raw.githubusercontent.com/citation-style-language/schema/master/schemas/styles/stop-words.json`

Comment (bwiernik) [reply specification.txt:2378]: Better: `https://resource.citationstyles.org/schema/latest/(.*)` to
`https://raw.githubusercontent.com/citation-style-language/schema/master/schemas/(.*)`

Comment (dstillman) [reply specification.txt:2378]: OK, redirects in place:

  * `schema/latest/input/json/` → `citation-style-language/schema/master/schemas/input/`
    * Example: https://resource.citationstyles.org/schema/latest/input/json/csl-citation.json
  * `schema/latest/` → `citation-style-language/schema/master/schemas/`
    * Example: https://resource.citationstyles.org/schema/latest/styles/stop-words.json

**Comment by bwiernik** [comment]
Closing this since it is now located at https://github.com/citation-style-language/schema/blob/master/schemas/styles/stop-words.json

**Comment by bwiernik** [comment]
@bdarcus Can you give me commit access and I will resolve and merge this?

**Pull request merged**